app/networkapi/people/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+class Person(models.Model):
+    name: models.CharField(max_length=120)
```

Comment:
How is this working? :P

it should be `name = models.CharField(max_length=120)`